application/layouts/scripts/parts/navigation.phtml

```diff
@@ -29,7 +29,3 @@ if (! $this->auth()->isAuthenticated()) {
         )
     ) ?>
 </div>
-<button id="toggle-sidebar" title="<?= $this->translate('Toggle Menu') ?>">
-    <i id="close-sidebar" class="icon-angle-double-left"></i>
-    <i id="open-sidebar" class="icon-angle-double-right"></i>
-</button>
```

Comment:
You mean, if it’s necessary to move it to menu.phtml?

I don’t think so. I just thought, it would be the right thing to do, since the toggle-sidebar button is now part ot the menu item.

But if you want, I can also reset it.

Reply:
Well, I've implied the justification for it being a part of `#menu`. It's a control of the sidebar, not of the menu after all.